packages/core/chrome/core-chrome-browser-internal/src/ui/project/header.tsx

```diff
@@ -70,6 +70,10 @@ const getHeaderCss = ({ size, colors }: EuiThemeComputed) => ({
     // needed to enable breadcrumbs truncation
     min-width: 0;
     flex-shrink: 1;
+    // prevents loading spinner from jumping
+    .eui-textTruncate {
+      display: flex;
```

Comment:
I don't love overwriting EUI styles, but this feels specific to this instance and not an issue with the EUI truncate utility.